SECURITY.md

```diff
@@ -0,0 +1,18 @@
+## Reporting Vulneratibilities
```

Comment:
It's good to have these instructions, but many users won't be looking for `SECURITY.md` when reporting a bug. How about having `BUGS.md` instead or in addition?